js/jquery.imageplayer.js

```diff
@@ -156,9 +158,8 @@ if (typeof(jQuery) == 'undefined') alert('jQuery library was not found.');
             if(settings.loop === true) {
                 if (i > images.length - 1) {
                     i = 0;
-                    // stop animation
-                    scrubber_handle.stop(true, true);
-                    scrubber_handle.css('left', '0');
+                    handle = document.getElementById('range');
```

Comment:
Instead of using document.getElementById and having to set a global id to the input element (setting a global id could collision with an id set by the webpage that uses this plug-in), you could do:

```js
var value = scrubber.val(); // to read the actual value
scrubber.val(i); // to set a value
```

scrubber is a global variable defined at the beginning of the file, the input tag "jQuery" element.
This way you can remove the id="" from the tag creation.

These 2 (161, 162) lines would become: `scrubber.val(0);`